### PR TITLE
Add gene search form to sample page.

### PR DESCRIPTION
**Pull request by rernst**
See feature request description in: 
https://dev.azure.com/umcu/LAB-Bioinformatics-Genetics/_backlogs/backlog/LAB-Bioinformatics-Genetics%20Team/Epics/?workitem=95983

### CODE REVIEW

File: ExonCov/views.py [hunk 1]
```diff
@@ -84,7 +108,7 @@ def sample(id):
                     [panels[panel.id]['len'], transcript_measurement.len]
                 )
             panels[panel.id]['len'] += transcript_measurement.len
-    return render_template('sample.html', sample=sample, panels=panels, measurement_types=measurement_types)
+    return render_template('sample.html', sample=sample, panels=panels, measurement_types=measurement_types, form=gene_form)
```
Comment (FiniDG) [inline ExonCov/views.py:111]: I don't really get the form reference here. 
in line 63, you have gene_form = SampleGeneForm(). so why is this new variable needed here?

Comment (rernst) [reply ExonCov/views.py:111]: `form` is the variable in the template `sample.html`, see: https://github.com/UMCUGenetics/ExonCov/blob/1b018ab3bed778adbe9f282a51315b26cde37427/ExonCov/templates/sample.html#L20 I typically call the 'form' variable 'form' in the flask templates to easily reuse code.

Comment (FiniDG) [reply ExonCov/views.py:111]: After discussion on Teams I get why it was made into a variable. You made the form in views.py and then called it in this script for usage. I think I missed that part. I understand it now.

**Review by FiniDG — APPROVED** [review]
Approved after discussion about one comment on the script.
Additionally I tested the functionality in the test.exoncov and compared it to the production.exoncov. It worked like expected.

**Pull request merged**